pioneer_control/PoseGrid pos
---
bool status
